mods/ts/rules/nod-structures.yaml

```diff
@@ -236,7 +236,7 @@ NATECH:
 		Cost: 2000
 	Tooltip:
 		Name: Nod Tech Center
-		Description: Required for high-\ntech research
+		Description: Required for high-tech research.
```

Comment:
I suggested an alternative for the GDI tech-center, which can be used here as well.